common/data/rumble.json

```diff
@@ -53987,6 +53987,431 @@
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1275 }
+            },
+            {
+              "override":  { "duration": 11 }
+            },
+            {
+              "override":  { "duration": 11 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1300 }
+            },
+            {
+              "override":  { "duration": 12 }
+            },
+            {
+              "override":  { "duration": 12 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1325 }
+            },
+            {
+              "override":  { "duration": 13 }
+            },
+            {
+              "override":  { "duration": 13 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1350 }
+            },
+            {
+              "override":  { "duration": 14 }
+            },
+            {
+              "override":  { "duration": 14 }
+            }
+          ]
+        },
+        {
+          "cooldown"; ,
+          "effects": [
+            {
+              "override":  { "amount": 1375 }
+            },
+            {
+              "override":  { "duration": 15 }
+            },
+            {
+              "override":  { "duration": 15 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1400 }
+            },
+            {
+              "override":  { "duration": 16 }
+            },
+            {
+              "override":  { "duration": 16 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1425 }
+            },
+            {
+              "override":  { "duration": 17 }
+            },
+            {
+              "override":  { "duration": 17 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1450 }
+            },
+            {
+              "override":  { "level": 2, "duration": 18 }
+            },
+            {
+              "override":  { "level": 2, "duration": 18 }
+            }
+          ]
+        },
+        {
+          "cooldown"; 34,
+          "effects": [
+            {
+              "override":  { "amount": 1700 }
+            },
+            {
+              "override":  { "level": 3, "duration": 19 }
+            },
+            {
+              "override":  { "level": 3, "duration": 19 }
+            }
+          ]
+        }
+      ],
+      "stats": {
+        "def": 215,
+	      "rumbleType": "SPT",
```

Comment:
fix indentation please

Reply:
indentation on my atom view is correct but once on github shows that way. I'll correct those things and add new characters in my next pull